Improved documentation of supported model types in LTR plugin (#8831)
opensearch-project/documentation-website: _search-plugins/ltr/training-models.md

Within the RankLib model, each tree in the ensemble examines feature values, makes decisions based on these feature values, and outputs the relevance scores. The features are referred to by their ordinal position, starting from 1, which corresponds to the 0th feature in the original feature set. RankLib does not use feature names during model training.

### Other RankLib models

> RankLib is a library that implements several other model types besides LambdaMART, such as MART,

Reviewer: ```suggestion
RankLib is a library that implements several other model types in addition to LambdaMART, such as MART,
```

> Notice that RankLib is a library that implements several other model types besides LambdaMART, such as MART,

Reviewer 2: ```suggestion
RankLib is a library that implements several other model types besides LambdaMART, such as MART,
```

> RankNet, RankBoost, AdaRank, Coordinate Ascent, ListNet or Random Forests. Each of these models has its own set of parameters and training process.

Reviewer 2: ```suggestion
RankNet, RankBoost, AdaRank, Coordinate Ascent, ListNet, and Random Forests. Each of these models has its own set of parameters and training process.
```

> For example, the RankNet model is a neural network that learns to predict the probability that a document is more relevant than another document. The model is trained using a pairwise loss function that compares the predicted relevance of two documents with the actual relevance. The model is serialized in a format similar to the following example:

Reviewer: ```suggestion
For example, the RankNet model is a neural network that learns to predict the probability of a document being more relevant than another document. The model is trained using a pairwise loss function that compares the predicted relevance of two documents with the actual relevance. The model is serialized in a format similar to the following example:
```

> For instance, the RankNet model is a neural network that learns to predict the probability that a document is more relevant than another document. The model is trained using a pairwise loss function that compares the predicted relevance of two documents with the actual relevance. The model is serialized in a format similar to the following example:

Reviewer 2: ```suggestion
For example, the RankNet model is a neural network that learns to predict the probability that a document is more relevant than another document. The model is trained using a pairwise loss function that compares the predicted relevance of two documents with the actual relevance. The model is serialized in a format similar to the following example:
```
